Formula/arduino-cli.rb

```diff
@@ -0,0 +1,24 @@
+class ArduinoCli < Formula
+  desc "Arduino command-line interface"
+  homepage "https://github.com/arduino/arduino-cli"
+  url "https://github.com/arduino/arduino-cli/archive/0.5.0-showports.tar.gz"
+  version "0.5.0-showports"
+  sha256 "96373b59e004f277474eb5a11e2a585633976c9939056b833e65e670e7393e77"
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/arduino/arduino-cli").install buildpath.children
+
+    cd "src/github.com/arduino/arduino-cli" do
+      system "go", "build", "-o", bin/"arduino-cli"
+      prefix.install_metafiles
+    end
```

Comment:
```suggestion
    system "go", "build", "-o", bin/"arduino-cli"
```

Seems like upstream uses Go modules, so it might be a good idea to take the advantage of that.

Reply:
Looks like multiline suggestions does not work.
Here is what I meant:
```
--- arduino-cli.rb.1	2019-10-06 22:40:20.398710823 +0200
+++ arduino-cli.rb	2019-10-06 22:38:10.391001070 +0200
@@ -8,13 +8,7 @@
   depends_on "go"
 
   def install
-    ENV["GOPATH"] = buildpath
-    (buildpath/"src/github.com/arduino/arduino-cli").install buildpath.children
-
-    cd "src/github.com/arduino/arduino-cli" do
-      system "go", "build", "-o", bin/"arduino-cli"
-      prefix.install_metafiles
-    end
+    system "go", "build", "-o", bin/"arduino-cli"
   end
 
   test do
```